src/GraphQl/Action/EntrypointAction.php

```diff
@@ -94,21 +94,17 @@ private function parseRequest(Request $request): array
         }
 
         if ('json' === $request->getContentType()) {
-            $input = json_decode($request->getContent(), true);
-
-            if (isset($input['query'])) {
-                $query = $input['query'];
-            }
+            return $this->parseInput($query, $variables, $operation, $request->getContent());
+        }
 
-            if (isset($input['variables'])) {
-                $variables = \is_array($input['variables']) ? $input['variables'] : json_decode($input['variables'], true);
-            }
+        if (false !== mb_stripos($request->headers->get('CONTENT_TYPE'), 'multipart/form-data')) {
```

Comment:
You should use the `getContentType` method.

Reply:
We can not use getContentType method because of here:
https://github.com/symfony/http-foundation/blob/master/Request.php#L1885
this function returns null.